pkg/scheduler/metrics/metrics.go

```diff
@@ -21,6 +21,7 @@ import (
 	"time"
 
 	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/kubernetes/pkg/controller/volume/persistentvolume"
```

Comment:
Any chance we could separate the metrics in a different package and import only the metrics package instead of the whole persistent volume controller?

Reply:
@wackxu would you be able to look into this?

Reply:
That sounds reasonable.